pyfr/libs/blueprint.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+from ctypes import c_void_p, c_long, RTLD_GLOBAL
+
+from pyfr.ctypesutil import LibWrapper
+from pyfr.libs.conduit import ConduitNode
+
+class BlueprintWrappers(LibWrapper):
+    _libname = 'blueprint'
+
+    # Functions
+    _functions = [
+        (c_long, 'conduit_blueprint_mesh_verify', c_void_p, c_void_p),
+        (c_long, 'conduit_blueprint_mcarray_verify', c_void_p, c_void_p),
+    ]
+
+    def __init__(self):
+        super().__init__(mode=RTLD_GLOBAL)
+
+class Blueprint:
```

Comment:
Do we need this?  There should be no way our code can generate invalid meshes.

Reply:
Sure, but as I had to code it so I could use it to debug, it made scene to commit it. Otherwise the next time someone wants to use these to debug they'll also have to implement this.

Reply:
Leave it in a branch.